### PR TITLE
Neutrontools

### PR DESCRIPTION
**Pull request by leczzsolt**
I included some tests. Probably the source data file (sdf) is also needed for running the tests...

### CODE REVIEW

**Comment by leczzsolt** [comment]
Hi Carsten !  Can you resolve the conflict in the readme file?  After that it can be merged into the develop branch.

**Review by CFGrote — CHANGES_REQUESTED** [review]
hi Zsolt,
looks excellent. Just have to add the tests to the unittest suite. See Tests/python/unittest/SimEx/Calculators/CalculatorsTests.py and Tests/python/unittest/SimEx/Parameters/ParametersTests.py how to do it. When you push, the new tests will be run on travis, let's see if everything goes through.

thanks!

File: Sources/python/SimEx/Calculators/TNSAIonMatterInteractor.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
+from SimEx.Calculators.AbstractIonInteractor import AbstractIonInteractor
+import sdf
```
Comment (JunCEEE) [inline Sources/python/SimEx/Calculators/TNSAIonMatterInteractor.py:2]: The travis error is: 

> ModuleNotFoundError: No module named 'sdf'

which is obviously related to this line. There is no `sdf` module found in the Travis environment. @leczzsolt is `sdf` a package which can be installed by `conda` or `pip`? If so, please add it into the [requirements.txt](https://github.com/PaNOSC-ViNYL/SimEx/blob/master/requirements.txt)

Comment (leczzsolt) [reply Sources/python/SimEx/Calculators/TNSAIonMatterInteractor.py:2]: The sdf package is added to the repository, but it is not in conda or pip (as far as I know).

Comment (JunCEEE) [reply Sources/python/SimEx/Calculators/TNSAIonMatterInteractor.py:2]: To solve the problem, the environment for `sdf` should be set for the Travis environment, @CFGrote do you know how to do it?

Hi @leczzsolt, I just noticed that you merged the branch without passing the test. The problem doesn't seem really solved. @CFGrote , I think we should pass the test before merging? Could you revert the merging?

**Comment by CFGrote** [comment]
$> echo SDF >> requirements.txt

--
Dr. Carsten Fortmann-Grote
PaNOSC WP5 leader

**Pull request merged**